internal/parse/parse.go

```diff
@@ -511,13 +517,18 @@ func getMounts(ctx *types.SystemContext, store storage.Store, mounts []string, c
 		if len(arr) < 2 {
 			return nil, mountedImages, lockedTargets, fmt.Errorf("%q: %w", mount, errInvalidSyntax)
 		}
-		kv := strings.Split(arr[0], "=")
+		typeToken := arr[0]
+		for _, field := range arr {
```

Comment:
`arr` has at most 2 elements here, so this loop doesn't iterate through all supplied arguments.

Reply:
I have changed `SplitN` with `Split`.